_maps/map_files/Event/laboratory.dmm

```diff
@@ -1996,6 +2038,11 @@
 /obj/effect/landmark/latejoin,
 /turf/open/floor/wood,
 /area/centcom)
+"HQ" = (
```

Comment:
Unfortunately due to this map edit this has to have the mapping tag. Good news is i dont think any other PR is editing the limbus labs map.